src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -56,438 +56,438 @@
 
 @SuppressWarnings("checkstyle:MethodName")
 public class ConfigurationPanelForm extends JComponent {
-  public JPanel container;
```

Comment:
You have change indentation on this file from 2 spaces to 4 spaces. Could you revert that, please? If you're using IntelliJ, there is a code style definition file in `config/codestyle/codestyle.xml`. You can import that to ensure this doesn't happen.

Reply:
Sure. I will do that